app/src/main/java/protect/card_locker/Utils.java

```diff
@@ -284,7 +285,10 @@ static public String formatBalance(Context context, BigDecimal value, Currency c
         currencyFormat.setMinimumFractionDigits(currency.getDefaultFractionDigits());
         currencyFormat.setMaximumFractionDigits(currency.getDefaultFractionDigits());
 
-        return currencyFormat.format(value);
+        // adding a space between the currency code and the balance
```

Comment:
Thanks. Though you're a bit too early for October :)

Are we sure there are no locales where the space should be on the left side? Or that otherwise behave differently in a way that matters here? And what about word wrapping?

I do agree "USD 5.00" probably looks nicer, but I think I'd prefer "€5.00" w/o the extra space, though not strongly.

Seems like a simple enough change, but we should make sure nothing unexpectedly breaks.

Reply:
I only adds a space for the currency codes -> USD 5.00...
.. but doesn't add it to the currency symbols -> £5.00
It depends on the locale, per example in French, we would write 5.00 £, and not £5.00.

Reply:
Right. I overlooked you're only replacing the codes. Makes sense.

That does mean that in French, you'd get `5,00 XXX ` with an extra space now. Maybe only replace it at the start then?

Reply:
Could adding `.trim()` to the end of  `return formattedCurrency.replace(currency.getCurrencyCode(), currency.getCurrencyCode() + " ")` remove the extra space ?